src/app/pages/password-reset/password-reset.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="parent">
+    <img src="../../../assets/img/Logo.png" alt="landville-logo" width="150" height="95">
+    <h1>Forgot your password?</h1>
+    <p>Enter your email address below and we'll get you back on track.</p>
+    
+    <form [formGroup]="resetForm" (ngSubmit)="onSubmit()">
+        <div class="form-group has-danger" [ngClass]="{'has-danger': resetForm.get('email').errors &&
+        (resetForm.get('email').touched || resetForm.get('email').dirty)}">
+          <input formControlName="email" placeholder="email address" class="form-control" ><br>
```

Comment:
Nice work @joelethan , though I have a concern of us having unnecessary calls to the backend to check for validations like in this case, you did not validate for the right email format